src/app.tsx

```diff
@@ -26,14 +28,23 @@ const App = ({ rep }: { rep: Replicache<M> }) => {
   const handleUpdateTodo = (update: TodoUpdate) =>
     rep.mutate.updateTodo(update);
 
-  const handleDeleteTodos = (ids: string[]) => {
-    rep.mutate.deleteTodos(ids);
+  const handleDeleteTodos = async (ids: string[]) => {
+    for (const id of ids) {
+      rep.mutate.deleteTodo(id);
```

Comment:
missing await... But you know that. I'm just worried about us silently ignoring errors.